docs/src/languages/nodejs/_index.md

```diff
@@ -128,33 +128,95 @@ npm prune --userconfig .npmrc && npm install --userconfig .npmrc
 
 This means you can specify configuration in a `.npmrc` file in [your app root](../../create-apps/app-reference.md#root-directory).
 
-### Use yarn as a package manager
+### Use Yarn as a package manager
 
-To switch to yarn to manage dependencies, follow these steps:
+To switch to Yarn to manage dependencies, follow these steps:
 
-1. Switch to a build flavor of `none` (so packages aren't installed with npm):
+1. Turn off the default use of npm:
 
    ```yaml {location=".platform.app.yaml"}
    build:
        flavor: none
    ```
 
-2. Add yarn as a global dependency:
+2. Specify the version of Yarn you want:
+
+   ```json {location="package.json"}
+   {
+     ...
+     "packageManager": "yarn@3.2.1"
+   }
+   ```
+
+What you do next depends on the version of Yarn and Node.js you want.
```

Comment:
:question:
Is it a typo?
Since the version used can be different for both Yarn and Node.js, wouldn't it be the plural form? (I'm not sure at all)

```suggestion
What you do next depends on the versions of Yarn and Node.js you want.
```

Reply:
I went back and forth on this for a while, to be honest. I guess if you also feel it, we can go with plural.